.github/workflows/deploy-to-lambda.yml

```diff
@@ -0,0 +1,30 @@
+name: deploy to lambda
```

Comment:
Also, for staging, the name of your file is `deploy-to-staging-lambda.yml`. Can you also renmae the prod file to something like `deploy-to-prod-lambda.yml`?